studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx

```diff
@@ -338,6 +350,46 @@ const ColumnEditor = ({
           />
         </FormSectionContent>
       </FormSection>
+
+      {isNewRecord && (
+        <>
+          <SidePanel.Separator />
+          <FormSection
+            header={<FormSectionLabel className="lg:!col-span-4">Security</FormSectionLabel>}
+          >
+            <FormSectionContent loading={false} className="lg:!col-span-8">
+              <Alert_Shadcn_>
+                <IconAlertCircle />
+                <AlertTitle_Shadcn_>
+                  Column encryption has been removed from the GUI
+                </AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_>
+                  <p className="!leading-normal">
```

Comment:
is the leading not normal by default ?

Reply:
its set to leading-relaxed for p tags, unless i should be using another tag here?

![image](https://github.com/supabase/supabase/assets/19742402/35e9f123-927b-49ce-b3b4-26d6e05cd7a2)